SETUP.md

```diff
@@ -258,11 +239,7 @@ Note: Virtual Machine Users should check the [Alternative note](#alternative-use
         1. `export CHROME_BIN=$(which chromium-browser)`
 1. Finally, configure your mysql to allow for a proper installation. You may run into errors if you did not leave mysql passwords blank
     1. `echo "ALTER USER 'root'@'localhost' IDENTIFIED WITH mysql_native_password BY '';" | sudo mysql`
-1. **IMPORTANT:** Read the following notes, then go back up to the [overview](#overview) and run the commands there.
-    1. If, for any reason, you are forced to interrupt the `bundle exec rake install` command before it completes,
-       cd into dashboard and run `bundle exec rake db:drop` before trying `bundle exec rake install` again
-    1. `bundle exec rake install` must always be called from the local project's root directory, or it won't work.
-    1. Finally, don't worry if your versions don't match the versions in the overview if you're following this method; the installation should still work properly regardless
+1. Return to the [overview](#overview) and continue setup.
```

Comment:
This note was not linux specific, but its a great tip: its been moved to the shared setup instructions as a troubleshooting tip for `bundle exec rake install`